internal/common/mqtt/mqttconnection_test.go

```diff
@@ -43,15 +49,44 @@ func TestStartMQTTClient(t *testing.T) {
 		err := StartMQTTClient(Host, "testClient", port)
 		expected := ""
 		if !strings.Contains(err, expected) {
-			t.Error("Unexpected err", err)
+			t.Error(unexpectedFail, err)
+		}
+		err1 := client.Subscribe("home/livingroom")
+		if err1 != nil {
+			t.Error(unexpectedFail, err1)
+		}
+		err = GetSubscribedData("home/livingroom", "testClient")
```

Comment:
It would be better to exclude the `mqttconnection` tc from this PR since it may collide with #544. Please update 'mqttconnection' tc after #544 is merged.

Reply:
Done!